packages/react-google-analytics/README.md

```diff
@@ -0,0 +1,68 @@
+# `@shopify/react-google-analytics`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-google-analytics.svg)](https://badge.fury.io/js/%40shopify%2Freact-google-analytics.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-google-analytics.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-google-analytics.svg)
+
+Allows React apps to easily embed tracking pixel iframes
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-google-analytics
+```
+
+## Usage
+
+This library exports a `<Universal />` & a `<GaJS />` component, which allows React apps to easily embed Google Analytics scripts.
+
+### Basic example
+
+```jsx
+import {
+  Universal,
+  GaJS,
+} from '@shopify/react-google-analytics';
+
+const UNIVERSAL_GA_ACCOUNT_ID = 'UA-xxxx-xx';
+const GA_JS_ACCOUNT_ID = 'UA-xxxx-xx';
+
+<Universal
```

Comment:
This is unclear - would one ever use both GaJS and Universal at the same time in an app? Or should users only import one that fits their need?

Which one is the recommended way?

Reply:
Good point, I guess I was focused on how we use it. We have to load both scripts in order to be on par with the way we do analytics in core.

I'll update the docs to just use or the other.

Thanks